src/plugins/vis_types/timeseries/public/convert_to_lens/lib/configurations/xy/layers.ts

```diff
@@ -170,38 +171,38 @@ const convertAnnotation = (
   annotation: Annotation,
   dataView: DataView
 ): EventAnnotationConfig | undefined => {
-  if (annotation.query_string) {
-    const extraFields = annotation.fields
-      ? annotation.fields
-          ?.replace(/\s/g, '')
-          ?.split(',')
-          .map((field) => {
-            const dataViewField = dataView.getFieldByName(field);
-            return dataViewField && dataViewField.aggregatable ? field : undefined;
-          })
-          .filter(nonNullable)
-      : undefined;
-    return {
-      type: 'query',
-      id: annotation.id,
-      label: 'Event',
-      key: {
-        type: 'point_in_time',
-      },
-      color: new Color(transparentize(annotation.color || euiLightVars.euiColorAccent, 1)).hex(),
-      timeField: annotation.time_field,
-      icon:
-        annotation.icon &&
-        ICON_TYPES_MAP[annotation.icon] &&
-        typeof ICON_TYPES_MAP[annotation.icon] === 'string'
-          ? ICON_TYPES_MAP[annotation.icon]
-          : 'triangle',
-      filter: {
-        type: 'kibana_query',
-        ...annotation.query_string,
-      },
-      extraFields,
-      isHidden: annotation.hidden,
-    };
-  }
+  const extraFields = annotation.fields
+    ?.replace(/\s/g, '')
+    ?.split(',')
+    .map((field) => {
+      const dataViewField = dataView.getFieldByName(field);
+      return dataViewField && dataViewField.aggregatable ? field : undefined;
+    })
+    .filter(nonNullable);
+
+  return {
+    type: 'query',
+    id: annotation.id,
+    label: 'Event',
+    key: {
+      type: 'point_in_time',
+    },
+    color: new Color(transparentize(annotation.color || euiLightVars.euiColorAccent, 1)).hex(),
+    timeField: annotation.time_field || dataView.timeFieldName,
+    icon:
+      annotation.icon &&
+      ICON_TYPES_MAP[annotation.icon] &&
+      typeof ICON_TYPES_MAP[annotation.icon] === 'string'
+        ? ICON_TYPES_MAP[annotation.icon]
+        : 'triangle',
+    filter: {
+      type: 'kibana_query',
+      /** Fixed difference in default behavior between visualizations.
```

Comment:
nit: I think this comment will be 'historic' very soon and I'd just remove it. The tests explain it enough.